src/styles/_catalog.scss

```diff
@@ -1,22 +1,32 @@
-.catalog {
-  display: flex;
-  flex-wrap: wrap;
+@mixin query ($cols) {
+    grid-template-columns: repeat($cols, 200px);
+}
 
-  max-width: 944px;
-  margin: 0 auto;
+.catalog {
+  display: grid;
+  justify-content: center;
+  align-content: center;
```

Comment:
No alignment needed here.
![image](https://user-images.githubusercontent.com/121403708/222897714-f4941d80-caf0-45c8-91a0-2822a4d68aa9.png)

Reply:
add  `align-content: start;` for `.catalog`